Installation not working on windows

### I am interested in helping provide a fix!

Yes

### Which generators are impacted?

- [X] All
- [ ] Angular
- [ ] HTML
- [ ] Preact
- [ ] Qwik
- [ ] React
- [ ] React-Native
- [ ] Solid
- [ ] Stencil
- [ ] Svelte
- [ ] Vue
- [ ] Web components

### Reproduction case

.

### Expected Behaviour

When running the command `npm run start` inside the library folder it should start watching files and generating components for the single frameworks

### Actual Behaviour

This does not happens on windows since it results in the following error:

```bash
PS C:\Users\giuli\Desktop\projects\ccc\library> npm run start

> start
> watch 'npm run build' ./src

> Watching run
> Watching build'
> Watching ./src
C:\Users\giuli\Desktop\projects\ccc\node_modules\watch\main.js:73
    if (err) throw err;
             ^

[Error: ENOENT: no such file or directory, stat 'C:\Users\giuli\Desktop\projects\ccc\library\run'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'stat',
  path: 'C:\\Users\\giuli\\Desktop\\projects\\ccc\\library\\run'
}

Node.js v20.13.1
npm ERR! Lifecycle script `start` failed with error:
npm ERR! Error: command failed
npm ERR!   in workspace: @ccc/library
npm ERR!   at location: C:\Users\giuli\Desktop\projects\ccc\library
```



### Additional Information

Turns out that the problem is in the `packaje.json` `start` script that uses single quotes for the command `npm run build`.
In windows terminal that does not work. The problem can be fixed by replacing the single quotes with escaped double quotes.
I'm wishing to open a PR to fix this.

--- a/package.json
+++ b/package.json
@@ -19,13 +19,13 @@
     "ci:test": "nx run-many --target test",
     "ci:e2e": "nx e2e @builder.io/e2e-app",
     "test:update": "nx run-many -t test:update",
-    "fmt": "run-s 'fmt:*'",
+    "fmt": "run-s \"fmt:*\"",
     "fmt:eslint": "yarn lint:eslint --fix",
     "fmt:prettier": "yarn lint:prettier --write",
-    "lint": "run-p -c 'lint:*'",
-    "lint:eslint": "eslint --cache 'packages/**/*.ts{,x}'",
+    "lint": "run-p -c \"lint:*\"",
+    "lint:eslint": "eslint --cache \"packages/**/*.ts{,x}\"",
     "lint:prettier": "yarn run prettier --check",
-    "prettier": "prettier --cache --loglevel warn '{packages,docs,e2e}/**/*.{js,jsx,ts,tsx,json,md,html}'"
+    "prettier": "prettier --cache --loglevel warn \"{packages,docs,e2e}/**/*.{js,jsx,ts,tsx,json,md,html}\""
   },
   "repository": {
     "type": "git",

